### ISSUE TITLE
[A11y][Reflow]Some of the contents under the select packages is getting overlapped when reflow is applied

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8190]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1188114
## Browser Details:
Firefox Version: 79.0 (64 bit)

## OS Details

Microsoft Windows 10 Enterprise  ​

Version 2004 (OS Build 19041.450) 

## Repro Steps:

    Change the Display resolution to 1280*1024 in System display settings
    Hit the URL and login with appropriate credentials
    Tab till "Open Profile drop down" and press enter
    Tab till "API Keys" and Press enter
    Tab till "Create button" and press enter
    Apply reflow
    Check whether data is overlapping is observed or not  

## Actual Results:

Some of the contents under the select packages is getting overlapped when reflow is applied 
## Expected Results:
User should be able to view the data without overlapping and all the information should be visible clearly with one scrollbar after applying reflow
## User Impact:
Users will not be able to understand the information if it gets overlapped in reflow

## Notes
This is in reference to the section in Create Key that shows example glob patterns. You do not have to change display resolution to see this. It is sufficient to shrink the window to minimum width and zoom in until the overlap occurs
![image](https://user-images.githubusercontent.com/11051729/92165397-d8e1de00-edeb-11ea-9bd8-7ce5bde30820.png)
